.gitignore

```diff
@@ -120,3 +120,6 @@ examples/tmp/
 
 # sandbox files
 _Untitled.*
+
+# results
+**/temp_output
```

Comment:
If you change this back to `tmp` as I suggested, then remember to change it also here